api/Text.json

```diff
@@ -0,0 +1,341 @@
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "Text": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Text/Text",
+          "description": "<code>Text()</code> constructor",
+          "support": {
+            "chrome": {
+              "version_added": "28"
+            },
+            "chrome_android": {
+              "version_added": "28"
+            },
+            "firefox": {
+              "version_added": "24"
+            },
+            "firefox_android": {
+              "version_added": "24"
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "15"
+            },
+            "opera_android": {
+              "version_added": "15"
+            },
+            "safari": {
+              "version_added": "8"
+            },
+            "safari_ios": {
+              "version_added": "8"
+            },
+            "webview_android": {
+              "version_added": "28"
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "isElementContentWhitespace": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Text/isElementContentWhitespace",
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": true,
+              "version_removed": "10"
+            },
+            "firefox_android": {
+              "version_added": true,
+              "version_removed": "10"
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": false
+            },
+            "opera_android": {
+              "version_added": false
+            },
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            },
+            "webview_android": {
+              "version_added": false
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": true
+          }
+        }
+      },
+      "wholeText": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Text/wholeText",
+          "support": {
+            "chrome": {
+              "version_added": "1"
+            },
+            "edge": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": true
+            },
+            "firefox_android": {
+              "version_added": true
```

Comment:
3.5

Reply:
@chrisdavidmills Travis doesn't like 3.5 as a number for Firefox.

Reply:
OK, just put 4 for now.